jib-core/src/main/java/com/google/cloud/tools/jib/registry/RegistryErrorExceptionBuilder.java

```diff
@@ -90,6 +92,9 @@ RegistryErrorExceptionBuilder addReason(String reason) {
   }
 
   RegistryErrorException build() {
+    // Provides a feedback channel.
+    errorMessageBuilder.append(
+        " | If this is a bug, please file an issue at https://github.com/google/jib/issues/new");
```

Comment:
Can you put `github.com/google/jib....` in a ProjectInfo file or something static that we reference everywhere?